### PR TITLE
Support language change from footer selector

### PR DESCRIPTION
**Pull request by alebedev**
## Justify why they are needed

Allows to test localization

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/blocks/FooterBlock.tsx [hunk 1]
```diff
@@ -79,9 +80,10 @@ export const FooterBlock = ({ blok }: FooterBlockProps) => {
     formRef.current?.dispatchEvent(new Event('submit', { cancelable: true, bubbles: true }))
   }
 
-  // todo: connect to locale / market
-  const onChangeLocale = (value: Locale | undefined) => console.log('Location changed:', value)
-
+  const router = useRouter()
+  const onChangeLocale = (locale: Locale | undefined) => {
+    router.push(router.asPath, undefined, { locale })
```
Comment (simonauner) [inline apps/store/src/blocks/FooterBlock.tsx:85]: Should we forward any query parameters? They will be removed with this, I guess

Comment (alebedev) [reply apps/store/src/blocks/FooterBlock.tsx:85]: Good catch, I'll handle this

Comment (alebedev) [reply apps/store/src/blocks/FooterBlock.tsx:85]: Already works.  If I change language on `/sv-se/about/why-hedvig?q=1` I'm redirected to `sv-ev/about/why-hedvig?q=1` -- all query parameters are retained

Comment (simonauner) [reply apps/store/src/blocks/FooterBlock.tsx:85]: Magic 🧙

**Review by simonauner — APPROVED** [review]

**Pull request merged**